zerver/views/webhooks/teamcity.py

```diff
@@ -0,0 +1,54 @@
+﻿# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.models import get_client
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+import ujson
+
+
+@api_key_only_webhook_view
+@has_request_variables
+def api_teamcity_webhook(request, user_profile, stream=REQ(default='teamcity')):
+
+    payload = ujson.loads(request.body)
```

Comment:
Ideally, this would have input validation similar to the stash.py webhook integration in this directory.